bin/cli.js

```diff
@@ -158,6 +158,10 @@ if (opts.slow) {
   mocha.slow(opts.slow)
 }
 
+if (opts.require) {
+  require(fs.absolute(opts.require));
+}
+
```

Comment:
For the help documentation it's easy enough to have something like the following:
`--modules        requires the given module or modules as a comma delimited string`
The README points to the mocha options and it's not exactly one of those. It could be moved to the section below with similar wording to above and an example such as `--modules=<file>,<file>`.